alacritty/src/display/mod.rs

```diff
@@ -487,14 +492,8 @@ impl Display {
             (Vec::new(), Vec::new())
         };
 
-        // We use vsync everywhere except wayland.
-        if !is_wayland {
-            if let Err(err) =
-                surface.set_swap_interval(&context, SwapInterval::Wait(NonZeroU32::new(1).unwrap()))
-            {
-                warn!("Error setting vsync: {:?}", err);
-            }
-        }
+        // Disable vsync.
+        let _ = surface.set_swap_interval(&context, SwapInterval::DontWait);
```

Comment:
I'm not sure how to handle failure of this. It could fail just because there's no extension. Maybe we should distinguish the failure from no extension and from the actual changing?

Reply:
I'm fine with ignoring it. We could log it maybe?